java/test/jmri/jmrix/loconet/LnThrottleManagerTest.java

```diff
@@ -187,13 +193,13 @@ public void notifyDecisionRequired(LocoAddress address, DecisionType question) {
                 "B5 11 10 00",
                 lnis.outbound.elementAt(6).toString());
 
-        JUnitUtil.waitFor(()->{return 6 < lnis.outbound.size();},"didn't get the 5th LocoNet message");
-        Assert.assertEquals("check count of sent messages", 6, lnis.outbound.size()-1);
+        JUnitUtil.waitFor(()->{return 6 < lnis.outbound.size();},"didn't get the 7th LocoNet message");
+        Assert.assertEquals("check count of sent messages", 7, lnis.outbound.size());
```

Comment:
There are a couple of changes I don't understand. For example, why is `lnis.outbound.size()` now 7 instead of 6?

Reply:
in the 1st line the comment was incorrect, the waitFor is for 7 outbound messages, not 5.
In the second line, the compare used to assert 6 = outboundmessages **-1** , now 7 = outboundmessages

Other changes to the tests are due, I think, to differences in Command Station used.

If someone with more Loconet experience wants to develop these tests further,
this should give them something stable to work with.

Reply:
Thanks! I missed `-1` in the original code.

Reply:
@icklesteve,
> Other changes to the tests are due, I think, to differences in Command Station used.

This test sets the command station as part of the setUp() method, and always configures for the DCS100.  Changing command station is not a factor!